ramp/core/stochastic_process.py

```diff
@@ -23,6 +22,13 @@ def calc_peak_time_range(user_list, peak_enlarg):
     rand_peak_enlarge = round(math.fabs(peak_time - random.gauss(peak_time, peak_enlarg * peak_time)))
     return np.arange(peak_time - rand_peak_enlarge , peak_time + rand_peak_enlarge)  # the peak_time is randomly enlarged based on the calibration parameter peak_enlarg
 
+def randomise_variables(var):
```

Comment:
I would add a second argument `norm` which is 1 by default. This way you can pass the value of for example `App.P_11` as norm and perform the multiplication inside the function, thus making the code even slicker

Reply:
Could you elaborate this a bit, please?